perl.h

```diff
@@ -1175,9 +1166,20 @@ violations are fatal.
 #    define USE_LOCALE
 #  elif defined(HAS_SETLOCALE)
 #    define USE_LOCALE
+#  else
+#    define NO_LOCALE
 #  endif
 #endif
 
+/* Even if not using locales, this header should be #included so as to #define
+ * some symbols which avoid #ifdefs to get things to compile.  But make sure
+ * the macro it calls does nothing */
+#undef PERL_LOCALE_TABLE_ENTRY
+#define PERL_LOCALE_TABLE_ENTRY(name, call_back)
+#include "locale_table.h"
+
+#include "perl_langinfo.h"    /* Needed for _NL_LOCALE_NAME */
+
```

Comment:
You are moving 9 lines of code here to be inside the section said to be `unfortunately duplicated in makedef.pl.`  However, I cant' find any of the following strings anywhere in `makedef.pl`.
```
$ grep -n -E '(PERL_LOCALE_TABLE_ENTRY|locale_table|perl_langinfo)' makedef.pl
$ 
```
Can you clarify?